server/modules/turf-review/turf-review.controller.ts

```
@@ -38,31 +45,215 @@ export default class TurfReviewController {
+    async (req: AuthRequest, res: Response): Promise<void> => {
+      const { userId } = req.params;
+
+      const options: ReviewFilterOptions = {
+        minRating: req.query.minRating
+          ? Number(req.query.minRating)
+          : undefined,
+        maxRating: req.query.maxRating
+          ? Number(req.query.maxRating)
+          : undefined,
+        limit: req.query.limit
+          ? parseInt(req.query.limit as string)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)